### DISCUSSION TITLE
Welcome to orcahome Discussions!

### DISCUSSION THREAD

**Discussion by scottveirs** [orcasound/orcahome#90] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by UXBrendan** [comment]
Hello! My name is Brendan, and I'm a UX Researcher by training. I function as the Principal UX Manager for the UX Team, as well as a Product Manager to help facilitate the process of sending design to production. Feel free to reach out and say hi!

**Comment by JacobYLiu** [comment]
Hi guys! I'm Jacob, and I'm a software engineer by training as well. Found orcasound in DemocracyLabs and I'm interested in learning more ways to help them! I want to contribute to the full stack of the project. Looking forward to collaborating with everyone.
